docs/guides/generic_recognizers.md

```diff
@@ -403,33 +406,34 @@ Post processing is optional - you may decide to combine or filter the "raw" even
-overlapping, then make use of the `SyllableSequence` parameter.
+Unlike overlapping events, if you want to combine a group of events (like syllables) that are near each other but not overlapping, then make use of the `SyllableSequence` parameter. A typical example would be to join a sequence of chirps in a honeyeater call.
 
 [!code-yaml[post_processing_combining_syllables](./Ecosounds.NinoxBoobook.yml?start=34&end=51&highlight=10- "Post Processing: Combining syllables")]
 
-Combining syllable sequences is enabled by setting `CombinePossibleSyllableSequence` true. A typical example is a sequence of chirps in a honeyeater call.
-
 `SyllableStartDifference` and `SyllableHertzGap` set the allowed tolerances when combining events into sequences
 
 - `SyllableStartDifference` sets the maximum allowed time difference (in seconds) between the starts of two events
 - `SyllableHertzGap` sets the maximum allowed frequency difference (in Hertz) between the minimum frequencies of two events.
 
-Once you have combined possible sequences, you may wish to remove sequences that do not satisfy the parameters for your
-target call. Set `FilterSyllableSequence` true if you want to filter (remove) sequences that do not fall within the
-constraints defined by `SyllableMaxCount` and `ExpectedPeriod`.
+> NOTE: In order to disable the combining of event sequences, "comment out" the `SyllableSequence` keyword by inserting the `#` symbol before it. You must also do the same for each of its five component parameters. 
+
+Once you have combined possible sequences, you may wish to remove sequences that do not satisfy the periodicity constraints for your target call, that is, the maximum number of syllables permitted in a sequence and the average time gap between syllables. 
+To enable filtering on syllable periodicity, set `FilterSyllableSequence` to true and assign values to `SyllableMaxCount` and `ExpectedPeriod`.
+
+- `SyllableMaxCount` sets an upper limit on the number of events that constitute an allowed sequence.
+- `ExpectedPeriod` sets an expectation value for the average period (in seconds) of an allowed combination of events.
+
+    > NOTE: When setting `ExpectedPeriod`, you are actually setting a permissible range of values for the period. The maximum permitted period will be the value assigned to `SyllableStartDifference` and the minimum period will be the `ExpectedPeriod` minus (`SyllableStartDifference` - `ExpectedPeriod`). For example: if `SyllableStartDifference` = 3 seconds and `ExpectedPeriod` = 2.5 seconds, then the minimum allowed period will be 2 seconds. 
 
-- `SyllableMaxCount` sets an upper limit of the number of events that are combined to form a sequence
-- `ExpectedPeriod` sets a limit on the average period (in seconds) of the combined events.
+    > NOTE: If you do not want to filter events on their periodicity, set `FilterSyllableSequence` to false. In this case, all events are accepted regardless of the periodicity of their component syllables.
```

Comment:
Note comments are syntactically incorrect.

Also comments about disabling should be removed and explained once in the post-processing section

Reply:
I have a lot of work here to insert lines, fix NOTE syntax and to shift unnecessary documentation to other guides.